packages/components/notification/__tests__/notification.vitest.ts

```diff
@@ -122,6 +122,9 @@ describe('Notification.vue', () => {
     })
 
     test('should not be able to render invalid type icon', () => {
+      const consoleWarn = console.warn
```

Comment:
I suggest to use `vi.spyOn`
You can refer to https://github.com/element-plus/element-plus/blob/b3684ce20678d462f232706ff05ee3133ee3fa3a/packages/components/form/__tests__/form-item.test.tsx#L54-L57